doc/examples/reconst_dki.py

```diff
@@ -0,0 +1,270 @@
+however you only need to fetch it once.
+"""
+
+from dipy.data import fetch_sherbrooke_3shell
+
+fetch_sherbrooke_3shell()
+
+"""
+Next, we read the saved dataset
+"""
+
+from dipy.data import read_sherbrooke_3shell
+
+img, gtab = read_sherbrooke_3shell()
+
+data = img.get_data()
+
+"""
+img contains a nibabel Nifti1Image object (with the data) and gtab contains a
+GradientTable object with information about the b-values and b-vectors. The
+b-values used on the loaded dataset are visualized above
+"""
+
+import matplotlib.pyplot as plt
+
+plt.plot(gtab.bvals, label='b-values')
+fig1 = plt.gcf()
+plt.legend()
+plt.show()
+fig1.savefig('HARDI193_bvalues.png')
+
+"""
+.. figure:: HARDI193_bvalues.png
+   :align: center
+   **b-values of the loaded dataset**.
+
+From the figure above, we can check that the loaded dataset contains three
+non-zero b-values as required for DKI. However the highest b-value of 3500
+$s.mm^{-2}$ is higher than normally used on DKI. Since DKI negletes diffusion
+signal components higher than the 4th order KT, a upper bound of b-value < 3000
+$s.mm^{-2}$ is normally implied to insure the fitting viability[Jensen2010]_.
+Following this, we discard the b-value shell of 3500 $s.mm^{-2}$ before DKI
+fitting
+"""
+
+select_ind = gtab.bvals < 3000
+selected_bvals = gtab.bvals[select_ind]
+selected_bvecs = gtab.bvecs[select_ind, :]
+selected_data = data[:, :, :, select_ind]
+
+"""
+The selected b-values and gradient directions are then converted to Dipy's
+GradientTable format.
+"""
+
+from dipy.core.gradients import gradient_table
+
+gtab = gradient_table(selected_bvals, selected_bvecs)
+
+"""
+Before fitting the data some data pre-processing is done. First, we mask and
+crop the data to avoid calculating Tensors on the background of the image.
+"""
+
+from dipy.segment.mask import median_otsu
+
+maskdata, mask = median_otsu(selected_data, 3, 1, True,
+                             vol_idx=range(10, 50), dilate=2)
+
+"""
+Now that we have prepared the datasets we can go forward with the voxel
+reconstruction. This can be done by first instantiate the DiffusinKurtosisModel
+in the following way.
+"""
+
+dkimodel = dki.DiffusionKurtosisModel(gtab)
+
+"""
+Fitting the data is very simple. We just need to call the fit method of the
+DiffusinKurtosisModel in the following way:
+"""
+
+dkifit = dkimodel.fit(maskdata)
+
+"""
+The fit method creates a DiffusionKurtosisFit object which contains all the
+diffusion and kurtosis fitting parameters and other DKI attributes. For
+instance, all DTI's standard tensor statistics can be computed from the
+DiffusionKurtosisFit instance as the fractional anisotropy (FA), the mean
+diffusivity (MD), the axial diffusivity (AD) and the radial diffusivity (RD).
+"""
+
+FA = dkifit.fa
+MD = dkifit.md
+AD = dkifit.ad
+RD = dkifit.rd
+
+"""
+Note that these DTI standard measures could also be computed from Dipy's DTI
+module. However, DTI models is only viable for b-values < 1500 $s.mm^{-2}$. For
```

Comment:
In what sense is DTI "only viable for b-values < 1500"? It provides a very accurate fit to the data even for 2k and 4k.